components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -626,6 +626,27 @@
   padding: 0.5em 0 0.5em;
 }
 
+.menu-box .patient-menu-button {
+  -webkit-border-radius: 2;
```

Comment:
`border-radius` is [well supported](http://caniuse.com/#feat=border-radius), so thete's no need for the prefixed proprties. Same for [`box-shadow`](http://caniuse.com/#feat=css-boxshadow).